examples/source_examples/aws_athena/using_sqlalchemy/connector.py

```diff
@@ -0,0 +1,79 @@
+from fivetran_connector_sdk import Connector
+from fivetran_connector_sdk import Operations as op
+
+TABLE_NAME = "test_rows"
+
+# Define the schema function which lets you configure the schema your connector delivers.
+# See the technical reference documentation for more details on the schema function:
+# https://fivetran.com/docs/connectors/connector-sdk/technical-reference#schema
+# The schema function takes one parameter:
+# - configuration: a dictionary that holds the configuration settings for the connector.
+def schema(configuration: dict):
+    return [
+        {
+            "table": "customers",  # Name of the table in the destination.
+            "primary_key": ["customer_id"],  # Primary key column(s) for the table.
+            "columns": {  # Define the columns and their data types.
+                "customer_id": "STRING",  # Integer column for the customer_id.
+                "first_name": "STRING",  # String column for the first name.
+                "last_name": "STRING",  # String column for the last name.
+                "email": "STRING",  # String column for the email.
+            },
+        }
+    ]
+
+def create_connection_engine(configuration):
+    conn_str = f"awsathena+rest://{configuration['aws_access_key_id']}:{configuration['aws_secret_access_key']}@athena.{configuration['region_name']}.amazonaws.com:443/{configuration['database_name']}?s3_staging_dir={configuration['s3_staging_dir']}"
+    return create_engine(conn_str)
+
+
+# Define the update function, which is a required function, and is called by Fivetran during each sync.
+# See the technical reference documentation for more details on the update function
+# https://fivetran.com/docs/connectors/connector-sdk/technical-reference#update
+# The function takes two parameters:
+# - configuration: dictionary contains any secrets or payloads you configure when deploying the connector
+# - state: a dictionary contains whatever state you have chosen to checkpoint during the prior sync
+# The state dictionary is empty for the first sync or for any full re-sync
+def update(configuration: dict, state: dict):
+    engine = create_connection_engine(configuration)
+
+    with engine.connect() as conn:
+        result = conn.execute(text(f"SELECT * FROM {TABLE_NAME}"))
+        while True:
+            rows = result.fetchmany(2)
+            if len(rows) == 0:
+                break
+
+            for row in rows:
+                yield op.upsert(table="customers",
+                                data={
+                                    "customer_id": row[0],  # Customer id.
+                                    "first_name": row[1],  # First Name.
+                                    "last_name": row[2],  # Last name.
+                                    "email": row[3],  # Email id.
+                                })
+
+
+# This creates the connector object that will use the update function defined in this connector.py file.
```

Comment:
```suggestion
# This creates the connector object - a connection - that will use the update function defined in this connector.py file.
```

Reply:
This is incorrect. When we say a connector object we mean the Connector Object in python code referred in the sdk package